joplin/templates/wagtailadmin/pages/edit.html

```diff
@@ -1,6 +1,85 @@
 {% extends "wagtailadmin/pages/edit.html" %}
 
+{% load wagtailadmin_tags %}
+{% load joplin_tags %}
+{% load i18n %}
+{% load l10n %}
+{% block titletag %}{% blocktrans with title=page.get_admin_display_title page_type=content_type.model_class.get_verbose_name %}Editing {{ page_type }}: {{ title }}{% endblocktrans %}{% endblock %}
+{% block bodyclass %}page-editor {% if page.live %}page-is-live{% endif %} model-{{ content_type.model }} {% if page.locked %}page-locked{% endif %}{% endblock %}
+
 {% block content %}
+    {% get_preview_url revision=page.get_latest_revision as preview_url %}
+    <script type="text/javascript">
+        function fallbackCopyTextToClipboard(text) {
```

Comment:
Test in browserstack and if we find incompatibilities then figure out a good design for backup

Reply:
Made an issue: https://github.com/cityofaustin/techstack/issues/544